packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -76,7 +77,9 @@ const AppConfig = ({children, locals = {}}) => {
 
 AppConfig.restore = (locals = {}) => {
     const locale = getLocale(locals) || DEFAULT_LOCALE
-    locals.api = new CommerceAPI({...apiConfig, locale})
+    const currency = getCurrency(locale)
+
+    locals.api = new CommerceAPI({...apiConfig, locale, currency})
```

Comment:
We previously made a request to the `commerce-sdk-isomorphic` team to support `locale` as a top-level configuration value for the api's. I don't see any reason not to make the same request to have `currency` also be a top-level config value. I'm going to leave a message for Ujwala Nandyala who's the PM on that team now. To see if we can get that on their radar.